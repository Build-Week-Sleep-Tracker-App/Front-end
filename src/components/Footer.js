import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../logo.png';
import { connect } from 'react-redux';
import { logout } from '../actions/actionCreators';

const Footer = props => {
	return (
		<footer>
			<div className="inner">
				<div className="logo">
					<img src={logo} alt="Logo" />
					<span>Sleep Tracker App</span>
				</div>
				<div className="moreInfo">
					<a href="https://awesome-sleep-tracker.netlify.com/" target="_blank" rel="noopener noreferrer">Marketing</a>
					<a href="https://build-week-sleep-tracker-app.github.io/Marketing-page-2/" target="_blank" rel="noopener noreferrer">Marketing 2</a>
				</div>
			</div>
		</footer>
	);
};

export default connect(
	state => ({ isLoggedIn: state.login.isLoggedIn }),
	null
)(Footer);