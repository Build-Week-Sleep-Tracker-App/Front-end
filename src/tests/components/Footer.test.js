import React from 'react';
import { shallow } from '../../enzyme';
import configureStore from 'redux-mock-store';

import Footer from '../../components/Footer';

const mockStore = configureStore();
const initialState = {
  login: {
    isLoggingIn: false,
    isLoggedIn: true,
    userId: null
  }
};
const store = mockStore(initialState);

const wrapper = shallow(<Footer store={store} />);

describe('Footer component tests', () => {
  test('should render footer component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
