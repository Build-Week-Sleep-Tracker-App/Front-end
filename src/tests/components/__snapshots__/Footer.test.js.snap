// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component tests should render footer component correctly 1`] = `ShallowWrapper {}`;
